games/rtype/include/System/ParallaxSystem.hpp

```diff
@@ -0,0 +1,20 @@
+/*
+** EPITECH PROJECT, 2023
+** RType
+** File description:
+** ParallaxSystem.hpp
+*/
+
+#ifndef RTYPE_PARALLAXSYSTEM_HPP
+#define RTYPE_PARALLAXSYSTEM_HPP
+
+#include "Components.hpp"
+#include "OtterCore.hpp"
+
+namespace Otter::Games::RType::System::Parallax {
+
+    void ParallaxMovement(Otter::Core::Orchestrator& ref);
```

Comment:
Need docs, i almose refuse cause i doen't understanded the purpose of this function